src/components/grid-aware/StatsBlock/StatsBlock.module.css

```
@@ -105,10 +105,7 @@
 
 .number {
   color: var(--color-sheltertech-blue);
-  font-family: var(--font-family-lato);
-  font-size: 72px;
-  font-weight: bold;
-  line-height: 1.3;
+  font: 300 72px/1.3 merriweather;
```

Could you use the variable that we have for Merriweather? This way, we have the fallback fonts set in case of this font doesn't load for some reason.

```suggestion
  font: 300 72px/1.3 var(--font-family-merriweather);
```